src/shared/components/MultiFileSelect/MultiFileSelect.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import { OverlayManagerProvider } from '@/hooks'
+import { Meta, Story } from '@storybook/react'
+import React, { useState } from 'react'
+import MultiFileSelect, { MultiFileSelectProps, FileState } from './MultiFileSelect'
+
+export default {
+  title: 'Shared/MultiFileSelect',
+  component: MultiFileSelect,
+  decorators: [
+    (Story) => (
+      <OverlayManagerProvider>
+        <Story />
+      </OverlayManagerProvider>
+    ),
+  ],
+} as Meta
+
+const Template: Story<MultiFileSelectProps> = (args) => {
+  const [files, setFiles] = useState<FileState>({
+    video: null,
+    image: null,
```

Comment:
After confirming thumbnail and then clicking and confirming again to change and choose new thumbnail - it doesn't change. Have to click for the third time to apply thumbnail change

Reply:
My intention was to have the ability to readjust **already uploaded image** after a click on the thumbnail.  I think it's more natural. Right now it doesn't work well, because file select also appears. I'll fix it soon